transcripts_expr_values_check.R

```diff
@@ -38,8 +41,19 @@ if (any(is.na(TPMexpr))) {
 
         # replace NAs with 0
         TPMexpr[is.na(TPMexpr)] <- 0
+        
+        # if any non-numeric column
+        if ( all(sapply(TPMexpr[,2:ncol(TPMexpr)], function(x){  is.numeric(x) })) == FALSE ){
+            print( "WARNING - at least one column has non-numeric values" ) 
```

Comment:
If this found will script run without failing?

Reply:
Yes. Although this is very unlikely, I have added instruction to exit the script if that happens [6f3fafc](https://github.com/ebi-gene-expression-group/atlas-analysis/pull/8/commits/6f3fafc71e45a519f32ac1403485ae402dc5ee48)